src/js/apps/patients/schedule/reduced-schedule/reduced_schedule_app.js

```diff
@@ -97,19 +97,43 @@ export default App.extend({
     if (!this.isRestarting()) this.stopChildApp('filters');
   },
   beforeStart() {
-    const filter = this.getState().getEntityFilter();
+    this.filters = this.getState().getEntityFilter();
     const fields = { flows: ['name', 'state'] };
-    return Radio.request('entities', 'fetch:actions:collection', { data: { filter, fields } });
+
+    return Radio.request('entities', 'fetch:actions:collection', { data: { filter: this.filters, fields } });
   },
   onStart(options, collection) {
     this.collection = collection;
     this.filteredCollection = collection.clone();
 
+    this.subscribe();
+
     this.listenTo(this.filteredCollection, 'reset', this.showCountView);
     this.showCountView();
 
     this.showList();
   },
+  subscribe() {
+    const channel = Radio.channel('ws');
+
+    channel.request('subscribe', this.collection.models, { filters: { actions: this.filters } });
+
+    this.listenTo(channel, 'message', ({ category }, model) => {
+      if (category === 'ActionDueChanged') {
+        this.showList();
```

Comment:
An `ActionDueChanged` ws notification can mean the action's due date or time changed. 

If the date changed, we need to move it to a different day list group. If the time changed, we only need to update the action's time and it doesn't need to be moved.

I'm not sure how to differentiate between those two scenarios yet.

But unlike the worklist & patient dashboard/flow pages, we need to handle this `ActionDueChanged` ws notification specifically.